### DISCUSSION TITLE
Launching apps with parameters

### DISCUSSION THREAD

**Discussion by rorlig** [doronz88/pymobiledevice3#769] [General]
I am able to launch google maps like so. 

python3 -m pymobiledevice3 developer dvt launch com.google.Maps --rsd fd3b:aad6:1b1::1 56657

I was wondering if there was a way send extra data to the app being opened e.g. 

https://stackoverflow.com/questions/32039816/how-to-open-google-maps-to-show-route-using-swift

allows an app to open google maps with driving directions.  Is it possible?

**Comment by rorlig** [comment]
any updates?

**Reply by doronz88** [reply]
if the app receives arguments via arg then yeah. it's not related to pymobile
